### DISCUSSION TITLE
无法转发加码动态

### DISCUSSION THREAD

**Discussion by muqiu791** [shanmiteko/LotteryAutoScript#323] [Q&A]
小白求教大佬们
版本是最新的2.8.11，无法识别并转发监视跟转的用户转发的加码动态，在监视的tag里加了“供电局福利社”也还是没用。这是为什么呢？
![image](https://github.com/shanmiteko/LotteryAutoScript/assets/108570297/577dead9-3a69-481e-beaf-6a183e562680)

**Comment by shanmiteko** [comment]
#152

**Reply by muqiu791** [reply]
”实现起来还是比较麻烦

监视tag其实就可以检测的那条b发的加码动态“。看了一下，还是不太明白监视tag怎么转发加码的。我的配置如下，在tag里加了”供电局福利社“，也加了参与TAGs，但还是没有转发，请问是其他的配置的问题吗。
![image](https://github.com/shanmiteko/LotteryAutoScript/assets/108570297/3c6aed8e-cdca-4d25-8fcb-355ae9683190)
